source/_patterns/02-molecules/vue-widgets/src/vue-cards/app.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div>
+    <h2>Vue Styles</h2>
+    <banner />
+    <cards
+      :cards="cardsArray"
+      @updateCards="update($event)"
+    />
+  </div>
+</template>
+
+<script>
+import cards from './components/cards.vue';
+import banner from './components/banner.vue';
+
+export default {
+  components: {
+    cards,
+    banner,
+  },
+  data() {
+    return {
+      cardsArray: [],
+    };
+  },
+  created() {
+    this.getCards();
+  },
+  methods: {
+    async getCards() {
+      this.cardsArray = await (await fetch(
+        'https://jsonplaceholder.typicode.com/users'
+      )).json();
+    },
+    update(eventId) {
+      // eventId starts at 1 we need the index which starts at 0.
+      const id = eventId - 1;
+      const target = this.cardsArray[id];
+      this.$set(target, 'isClicked', !target.isClicked);
```

Comment:
This is a great start, but we actually want the `isClicked` value to be the decider of highlight class also.